packages/eslint-plugin/src/rules/dependency-checks.ts

```diff
@@ -350,13 +341,9 @@ export default createESLintRule<Options, MessageIds>({
         }
 
         if (expectedDependencyNames.includes(packageName)) {
-          return validateVersionMatchesInstalled(
-            node,
-            packageName,
-            packageRange
-          );
+          validateVersionMatchesInstalled(node, packageName, packageRange);
         } else {
-          return reportObsoleteDependency(node, packageName);
```

Comment:
We should either remove `return` everywhere or keep it as it was.

Reply:
Didn't seem necessary since the return  was void, but I see it's the case in the other functions too. I'll update.

Reply:
Thanks